package.json

```
@@ -250,7 +250,18 @@
           "description": "An array of RegExps to use as TODO for note suggestions"
         }
       }
-    }
+    },
+    "colors": [
+      {
+        "id": "codeAnnotaion.mainColor",
+        "description": "Main color.",
```

I think a better description would be:

```suggestion
        "description": "Color for the background for the annotation on the source code",
```